### ISSUE TITLE
accessing django-admin and cyclops on the same port

### ISSUE THREAD

**Issue by adamstauffer** [dunbarcyber/cyphondock#9]
Something that isn't very clear in the docs - in the current configuration if you want to get to the admin panel you need to hit directly on port 8000/tcp (http://hostname:8000/admin), meaning that this port also needs to be exposed on the host. 

For a more seamless experience, I wanted to access the admin panel over port 80 (meaning http://hostname/app would go to cyclops and http://hostname/admin would go to cyphon django-admin). This required the following:

* nginx configuration to proxy_pass anything to ^/admin.*$ to cyphon. Adding this to the server 80 block seemed to work:
```
location ~ ^/admin(.*)$ {
    proxy_pass http://cyphon:8000;
}
```
* changing STATIC_URL and STATIC_ROOT in the django settings. Unfortunately this is necessary because cyclops and the current django configuration both try to serve out of /static. I changed django serve as `/usr/src/app/static/` and moved the nginx static block to the server 80 block.

Thus, my the applicable sections of my nginx configuration looks like this:

```
    server {
        listen 80;
        server_name localhost;
        charset utf-8;

        access_log /dev/stdout;
        error_log /dev/stdout warn;

	location ~ ^/admin(.*)$ {
            proxy_pass          http://cyphon:8000;
	}

        location /usr/src/app/static/ {
            alias /usr/src/app/static/;
            include /etc/nginx/mime.types;
        }

        location / {
            proxy_pass          http://cyclops:3000;
            proxy_set_header    Host $host;
            proxy_set_header    X-Real-IP $remote_addr;
            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header    X-Forwarded-Host $server_name;
        }
    }

    server {
        listen 8000;
        server_name localhost;
        charset utf-8;

        access_log /dev/stdout;
        error_log /dev/stdout warn;

        location / {
            proxy_pass          http://cyphon:8000;
            proxy_set_header    Host $host;
            proxy_set_header    X-Real-IP $remote_addr;
            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header    X-Forwarded-Host $server_name;
        }

        # deny access to Passport key files
        location /keys/ {
           deny all;
           return 404;
        }

    }
```

I wouldn't really consider this a bug but more of an improvement (either in the docs or formalizing this kind of solution in these config files). Another option would be publishing the django-admin static content with cyclops (assuming there are no conflicts).

**Comment by lhadjchikh** [comment]
Great suggestion, thanks! That's definitely better than pointing to port 8000.

I'll see about incorporating this into our official Cyphondock release (which should be soon). I also have another `nginx.conf` that uses subdomains for the various services instead. I might provide that as an example for those who want to use subdomains, but I like your method.

**Comment by lhadjchikh** [comment]
This should be fixed in the [1.2 release](https://github.com/dunbarcyber/cyphon/releases/tag/1.2.0). There's a brief summary in our [blog post](https://www.cyphon.io/blog/2017/6/5/cyphon-release-12). Let us know if you any have questions!
